components/menus/menu-list/readme.md

```diff
@@ -0,0 +1 @@
+A sort title, when clicked, expands a list of links.
```

Comment:
It would be nice to flesh out this documentation with some more description of how it works, suggestions on uses, mention how to use the JS to initialize it, etc. 

Not a blocker for merging since we can always add more later, but this is pretty scant documentation (I know it was just taken from the original; that one should have been better documented too).